scripts/usv_master.py

```diff
@@ -71,19 +86,19 @@ def main():
         elif (usvMaster.xbee_message)== "AN" or  (usvMaster.xbee_message)== "an":
             current_node = subprocess.Popen("rosrun rb_missions auto_nav_position.py", shell = True)
             usvMaster.xbee_message= ""
-            print(usvMaster.xbee_message)
-            while usvMaster.mission_status != 1 and  usvMaster.stop_mission!=True and not rospy.is_shutdown():
+            while usvMaster.mission_status != 1 and  usvMaster.kill_switch!=True and not rospy.is_shutdown():
```

Comment:
Readability: condiciones agrupadas en paréntesis.
(usvMaster.mission_status != 1) and (...) and not (...)